lib/Service/SettingsService.php

```diff
@@ -63,6 +63,12 @@ class SettingsService {
 	const DEFAULT_MINIMUM_CONFIDENCE = '0.99';
 	const MAXIMUM_MINIMUM_CONFIDENCE = '1.1';
 
+	/** Minimum face size used to try to clustring faces */
+	const MINIMUM_FACE_SIZE_KEY = 'min_face_size';
+	const MINIMUM_MINIMUM_FACE_SIZE = '0';
+	const DEFAULT_MINIMUM_FACE_SIZE = '125';
```

Comment:
Why do you do this, because small faces tend to give wrong result/bad 128D descriptors? Or to remove non-important stuff/features from image? If former, it makes sense to have this as absolute value (in pixels). But, if latter, maybe you should define `min_face_`size` as percentage from image size? Just saying, you probably though of this more than me:D

Reply:
> Why do you do this, because small faces tend to give wrong result/bad 128D descriptors?

Yes. I use 125 instead of 150 because I assume 125 are still good faces. But the famous 99.34% quality of the model is with 150px faces and good quality.

Note that there are many photos, which seems good at a glance, but if you only see the detail of the amplified face rectangle, you will see that it is not so easy to recognize a person.
So, at first glance, Us we recognize it from the context of the photo, but with only the rectangle, it is difficult to determine.
Well, the model has a similar problem, if you see a 50x50 face, it doesn't have enough information to determine a good descriptor.

This is the idea, but as I said we have not yet addressed the problem of resizing photos. You can still get the descriptor of a large face, on a much smaller rectangle, and theoretically its descriptor would not be good either.

However, I think we will agree that a 50px face is not good to compare. :sweat_smile: